stock_production_lot_active/models/stock_production_lot.py

```diff
@@ -1,21 +1,10 @@
 # Copyright 2020 ACSONE SA/NV
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import api, fields, models
+from odoo import fields, models
 
 
 class StockProductionLot(models.Model):
-    _inherit = "stock.production.lot"
+    _inherit = "stock.lot"
 
-    active = fields.Boolean(default=True)
-
-    @api.constrains("name", "product_id", "company_id")
-    def _check_unique_lot(self):
```

Comment:
@laszlokr Why removing this?

Reply:
@rousseldenis this check seems to be redundant, as by default it is not possible to have the same product with identical lot/serial number. I tested the module with and without this method, the behavior is exactly same

Reply:
Have you tested with inactive lots ?

As in standard, it seems inactive lots are not tested: 

![image](https://github.com/user-attachments/assets/a5cf0a7c-27ab-4ba9-908f-92391bc2298a)

Reply:
@rousseldenis apologies for my delayed response. Indeed, it might still be needed, so I restored the method and also renamed the class according to the change of model name